python/cuml/svm/svm_base.pyx

```diff
@@ -234,7 +234,8 @@ class SVMBase(Base,
 
         # Attributes (parameters of the fitted model)
         self.dual_coef_ = None
-        self.support_ = None
+        #self.support_ = None
+        self._support_ = None
```

Comment:
I’m not entirely sure if this variable name change is needed, the getters and setters should still work without the rename, no? (heven’t tried running the code locally)